mailer.rst

```diff
@@ -111,13 +111,18 @@ Postmark            ``composer require symfony/postmark-mailer``
 SendGrid            ``composer require symfony/sendgrid-mailer``
 Sendinblue          ``composer require symfony/sendinblue-mailer``
 MailPace            ``composer require symfony/mailpace-mailer``
+Infobip             ``composer require symfony/infobip-mailer``
 ==================  ==============================================
 
 .. versionadded:: 6.2
 
     The ``MailPace`` integration was introduced in Symfony 6.2 (in previous
     Symfony versions it was called ``OhMySMTP``).
 
+.. versionadded:: 6.2
```

Comment:
Maybe it should be merged with the previous block?

Reply:
I left them separate for now because the existing `versionadded` is a bit special and explains some renaming. Merging both would look confusing in my opinion. But we can change this at any moment if we disagree. Thanks!